tempesta_fw/apm.c

```diff
@@ -0,0 +1,1039 @@
+/**
+ *		Tempesta FW
+ *
+ * Copyright (C) 2016 Tempesta Technologies, Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License,
+ * or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, write to the Free Software Foundation, Inc.,
+ * 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA.
+ * Prototype for fast precentiles calculation.
+ */
+#include <linux/kernel.h>
+#include <linux/slab.h>
+#include <linux/atomic.h>
+#include <linux/spinlock.h>
+#include <linux/stringify.h>
+#include <linux/sort.h>
```

Comment:
It seems `sort.h` isn't needed any more. Also please keep includes in alphabetical order - it's always easier to find an include in ordered sequence.